model/gen-ai/spans.yaml

```diff
@@ -119,6 +119,8 @@ groups:
         requirement_level: required
       - ref: gen_ai.request.top_k
         requirement_level: recommended
+      - ref: gen_ai.tool_definitions.raw
```

Comment:
I think it should go to the `attributes.gen_ai.inference.client` group and it would populate it on all inference spans, agent spans, and also details event (which mirrors inference span).

Reply:
added the `gen_ai.tool_definitions` attribute to `attributes.gen_ai.inference.client`. Pushing the change in next PR. Thanks @lmolkova.